ww-config.js

```diff
@@ -614,21 +539,17 @@ export default {
             },
             /* wwEditor:end */
         },
-        borderRadius: {
-            label: { en: 'Border Radius' },
-            type: 'Length',
-            section: 'style',
-            bindable: true,
-            defaultValue: '0px',
-            /* wwEditor:start */
-            bindingValidation: {
-                type: 'string',
-                tooltip: 'The border radius of the breadcrumbs container',
-            },
-            propertyHelp: {
-                tooltip: 'Set the border radius for the breadcrumbs container',
-            },
-            /* wwEditor:end */
+        linkElement: {
+            hidden: true,
+            defaultValue: { isWwObject: true, type: 'ww-flexbox' },
+        },
+        iconElement: {
+            hidden: true,
+            defaultValue: { isWwObject: true, type: '1b1e2173-9b78-42cc-a8ee-a6167caea340' },
```

Comment:
@MarcFabre at the end, do we change the alias so that could be replace by ww-icon?

Reply:
the alias ww-icon is now the new ww-icon element, we can you it there.

Reply:
I renamed the type to `ww-icon`